_includes/sxs/02-extending/testimonial.html

```diff
@@ -1,19 +1,17 @@
-<div class="usa-card ut-testimonial">
-  <div class="usa-card__container">
-    <svg class="usa-icon ut-testimonial__icon" aria-hidden="true" focusable="false" role="img" height="50px" width="50px">
-      <use xlink:href="/assets/uswds/img/sprite.svg#format_quote"></use>
-    </svg>
-    <div class="usa-card__body ut-testimonial__quote">
-      <p>
-        "Wow, this component sure is great"
-      </p>
-    </div>
-    <div class="usa-card__footer ut-testimonial__footer">
-      <img src="https://doodleipsum.com/100x100/avatar-5?bg=a5a8eb&shape=circle&n=1" alt="Profile pic">
-      <div class="ut-testimonial__profile">
-        <p class="ut-testimonial__profile-name">Person McGuy</p>
-        <p class="ut-testimonial__profile-title">USWDS User</p>
-      </div>
+<div class="ut-testimonial">
+  <svg class="usa-icon ut-testimonial__icon" aria-hidden="true" focusable="false" role="img" height="50px" width="50px">
+    <use xlink:href="/assets/uswds/img/sprite.svg#format_quote"></use>
+  </svg>
+  <div class="ut-testimonial__quote">
+    <p>
+      "Wow, this component sure is great"
+    </p>
+  </div>
+  <div class="ut-testimonial__footer">
+    <img src="https://doodleipsum.com/100x100/avatar-5?bg=a5a8eb&shape=circle&n=1" alt="Profile pic">
+    <div class="ut-testimonial__profile">
+      <p class="ut-testimonial__profile-name">Person McGuy</p>
+      <p class="ut-testimonial__profile-title">USWDS User</p>
     </div>
   </div>
-</div>
+</div>
```

Comment:
Can we add the line at the end?

Reply:
Resolved in [1164c5b](https://github.com/uswds/uswds-tutorial/pull/32/commits/1164c5b6f7559432880998529ab7679ff722f2c0)